sites/diverseeducation.com/config/navigation.js

```diff
@@ -50,6 +50,8 @@ const utilities = [
 const mobileMenu = {
   primary: [
     ...topics,
+    ...resources,
```

Comment:
@jwade1327 I think we can make the case (at least for Diverse Education) to put "Resources" under secondary here prior to subscribe etc. RR/Overdrive does something similar it makes it so users don't have to scroll nearly as much in order to see everything going on and everything will still "be there"